js/views/userPage/UserPageFollow.js

```diff
@@ -1,23 +1,104 @@
-import loadTemplate from '../../utils/loadTemplate';
-import { View } from 'backbone';
+import BaseVw from '../baseVw';
+import userShort from '../userShort';
+import app from '../../app';
+import Follows from '../../collections/Followers';
+import followUtils from '../../utils/follow';
 
-export default class extends View {
+export default class extends BaseVw {
   constructor(options = {}) {
-    super({
-      className: 'userPageFollow',
-      events: {
-      },
-      ...options,
-    });
+    super(options);
+    this.options = options;
+    this.followType = options.followType;
+
+    if (!options.ownPage) {
+      this.followCol = new Follows(null, {
+        type: this.followType.toLowerCase(),
+        guid: this.model.id,
+      });
+      this.followCol.fetch().done(() => {
+        this[`updateViewer${this.followType}`](true);
+        this.render();
+      });
+
+      this.listenTo(app.ownFollowing, 'sync, update', () => {
+        this.updateViewerFollowers();
+        this.render();
+      });
+
+      this.listenTo(app.ownFollowers, 'update', () => {
+        this.updateViewerFollowing();
+        this.render();
+      });
+    } else {
+      this.followCol = app[`own${this.followType}`];
+      if (this.followType === 'Followers') {
+        this.followCol.fetch().done(() => {
+          this.render();
+        }); // TODO: pagination
+      } else {
+        this.render();
+      }
+
+      this.listenTo(app[`own${this.followType}`], 'sync, update', () => {
+        this.render();
+      });
+    }
   }
 
-  render() {
-    loadTemplate('userPage/userPageFollow.html', (t) => {
-      this.$el.html(t({
-        ...this.model.toJSON(),
-      }));
-    });
+  updateViewerFollowers() {
+    /* if the viewer follows/unfollows this user, add them to the followers list without a fetch */
+    if (this.followType === 'Followers' && !this.options.ownPage) {
+      // if the viewer has followed add them
+      if (followUtils.followedByYou(this.model.id)) {
+        this.followCol.add({ guid: app.profile.id });
```

Comment:
Once again, if I'm not mistaking about most recent followers being at the top of the list, then we want Collection.unshift() here.